GitSearch/Networking/HTTP/HTTPNetworkRequest.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  HTTPNetworkRequest.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/27/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+public typealias HTTPParameters = [String: Any]?
+public typealias HTTPHeaders = [String: Any]?
```

Comment:
It's probably better not to make this types optional - you can add optionality to a function parameters instead:
`func foo(_ params: HTTPParameters?)`
Right now you can consider `nil` as `HTTPParameters` (doubtful), but if you have optional parameter, not a type, than it's obvious that `nil` means parameter can just be omitted.

Reply:
Yes, you are right.
Even in function configureParametersAndHeaders I made double optional declaration: had optional type and optional params